### DISCUSSION TITLE
Why has development paused?

### DISCUSSION THREAD

**Discussion by kapple19** [MichaelHatherly/Publish.jl#65] [General]
Hi all,

I'm relatively novice to Julia. I'm looking for a publishing solution that satisfies the following features:

* Large scale document structuring with links and referencing for sections, figures, tables, etc.
* TeX display with custom macros.
* Reactive source code implementation.

I like Publish.jl most out of available Julia package options because:

* It focuses on the published content, unlike Documenter.jl which is centred on the package. I just want to make notes with the source code in the notes.
* I think Publish.jl has a smoother server update, unlike Documenter.jl.
* It looks like it could support TeX macros, unlike Pluto.jl. AFAIU, this isn't support (yet?) in Publish.jl.
* The Markdown syntax is much, much more preferable than Franklin.jl.

I'm willing to help with Publish.jl, it'll just be all new to me. I just thought I'd ask if people migrated to other static static site generators for Julia.

Thanks in advance!

**Comment by MichaelHatherly** [comment]
Hi @kapple19, it's simply due to lack of time on my part that development has stalled. Short-term I'm unlikely to be able to allocate enough time to push development along unfortunately. Happy to guide PRs and features though wherever needed.

**Reply by kapple19** [reply]
@MichaelHatherly that's totally understandable, and in addition to my novice experience in Julia, my contribution to this package would be minimal due to a lack of time as well.
